### PR TITLE
add protocol and framework specmarkdown

### PR DESCRIPTION
**Pull request by yuriko627**
moved hackmd file github

### CODE REVIEW

**Comment by daiki44** [comment]
memo: https://deploy-preview-67--quirky-chandrasekhar-8f7e92.netlify.app/docs/protocol-and-framework-spec/protocol-and-framework-spec/

File: pages/docs/protocol-and-framework-spec/protocol-and-framework-spec.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+- **Dispute period**: period that the dispute game is played. Users have to wait this period to withdraw their assets.
+
+### Scaling solutions powered by fraud-proof scheme
+
+### Plasma 　
+
+Plasma is a blockchain scaling design which significantly reduces the gas cost per transaction.
+Using Plasma, transactions are not directly submitted to the mainchain, but to a **transaction aggregator**, also known as an **operator**, who later submits them hashed into 32 bytes data. Aggregator computes the root hash of a Merkle tree in which every leaf node is labeled with a hash of state from off-loaded transactions and commits the root hash to the mainchain.
+
+![](https://i.imgur.com/bAqdgPc.png)
```
[the remark this answers is not in the document]
Comment (yuriko627) [reply pages/docs/protocol-and-framework-spec/protocol-and-framework-spec.md]: moved them

File: pages/docs/protocol-and-framework-spec/protocol-and-framework-spec.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+---
+id: protocol-and-framework-spec
```
[the remark this answers is not in the document]
Comment (yuriko627) [reply pages/docs/protocol-and-framework-spec/protocol-and-framework-spec.md]: For now, I changed the ID to Protocol_And_Framework_Spec as you suggested.

File: pages/docs/protocol-and-framework-spec/framework-spec.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+id: Gazelle-Documentation
```
Comment (daiki44) [inline pages/docs/protocol-and-framework-spec/framework-spec.md:2]: `-` -> `_`

File: pages/docs/protocol-and-framework-spec/protocol-spec.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
+---
+id: Protocol-Spec
```
Comment (daiki44) [inline pages/docs/protocol-and-framework-spec/protocol-spec.md:2]: `-` -> `_`

File: pages/docs/protocol-and-framework-spec/protocol-spec.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
+---
+id: Protocol_Spec
+title: Protocol Spec
+sidebar_label: Protocol Spec
+---
+
+**Introduction**
```
Comment (daiki44) [inline pages/docs/protocol-and-framework-spec/protocol-spec.md:7]: I guess `**Introduction**` might be `## Introduction`.
https://deploy-preview-67--quirky-chandrasekhar-8f7e92.netlify.app/docs/protocol-and-framework-spec/protocol_spec/

**Pull request merged**